README.md

```diff
@@ -1,35 +1,54 @@
-<img src="https://raw.githubusercontent.com/onc-healthit/inferno/master/public/images/inferno_logo.png" width="300px" />
+<img
+src="https://raw.githubusercontent.com/onc-healthit/inferno/master/public/images/inferno_logo.png"
+width="300px" />
 
-[![Build Status](https://travis-ci.org/onc-healthit/inferno.svg?branch=master)](https://travis-ci.org/onc-healthit/inferno)
+[![Build
+Status](https://travis-ci.org/onc-healthit/inferno.svg?branch=master)](https://travis-ci.org/onc-healthit/inferno-program)
 
-Inferno is an open source tool that tests whether patients can access their health data through a standard interface.
-It makes HTTP(S) requests to test your server's conformance to authentication, authorization, and FHIR content standards and reports the results back to you.
+# Inferno for mCODE
 
-This application creates test clients that exercise the range of requirements of a [SMART on FHIR](http://smarthealthit.org/smart-on-fhir/) server. These clients have tests for the [SMART App Launch Framework](http://www.hl7.org/fhir/smart-app-launch/), [Argonaut](http://hl7.org/fhir/DSTU2/argonaut/argonaut.html) Implementation Guides, and [FHIR DSTU2](http://hl7.org/fhir/DSTU2/index.html).
+## The Situational Awareness for Novel Epidemic Response Implementation Guide Testing
```

Comment:
anything SANER related should be removed.  Please take another sweep through the repo.